infrastructure/proto/log_replication_cluster_info.proto

```diff
@@ -1,35 +0,0 @@
-syntax = "proto3";
-
-package org.corfudb.infrastructure.logreplication.proto;
-
-// Multi-Cluster Topology configuration
-message TopologyConfigurationMsg {
```

Comment:
These protobufs were never persisted, right?  Otherwise they will have to be handled during rolling upgrade

Reply:
No, they weren't